src/AppInstallerCommonCore/Public/AppInstallerProgress.h

```diff
@@ -31,9 +31,16 @@ namespace AppInstaller
     // of cancel state.
     struct IProgressSink
     {
-        // Called as progress is made.
+        // Called as progress is made within the current execution stage.

```

Comment:
>within the current execution stage. [](start = 38, length = 35)

This progress infrastructure has no concept of execution stage, and I feel that adding these comments make it seem like a single progress is tied to a single execution stage, which is not true.